src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/CompletionHandlerTest.cs

```diff
@@ -233,7 +233,7 @@ public async Task HandleRequestAsync_CSharpProjection_DoNotPreselectAfterAt()
         {
             // Arrange
             var called = false;
-            var expectedItem = new CompletionItem() { InsertText = "__builder", Label = "__builder", Preselect = true };
+            var expectedItem = new CompletionItem() { InsertText = "AccessViolationException", Label = "AccessViolationException", Preselect = true };
```

Comment:
Technically `_Imports` is the first item in the list now that `__builder` is removed, but I changed this to `AccessViolationException` since `_Imports` technically shouldn't be in the list either (https://github.com/dotnet/aspnetcore/issues/29562).